### ISSUE TITLE
Add Nuget support

### ISSUE THREAD

**Issue by VadymSV** [Appboy/appboy-xamarin-bindings#3]
Please add your SDK's to Nuget. It will be much easier to manage SDK via Nuget and do not have binary in my repository.

**Comment by timahrentlov** [comment]
Like Urban Airship that now has a .NET Core nuget. I mean, come on guys.

**Comment by Bucimis** [comment]
Hi @VadymSV,

Thanks for filing this issue! By any chance, are you able to use the Xamarin Component Store (available in Xamarin Studio and Visual Studio)?  Our Xamarin package is available there - also note that components from the Xamarin Component Store do not need to be checked into source control.

Thanks,
Appboy Xamarin Team

**Comment by VadymSV** [comment]
Hi @Bucimis,

I think it a rule of game for big companies to have Nuget package of their product if they support .NET. For me it's not a big deal to replace *.dlls but nevertheless. For now this repository looks very poor... Please pay attention on it.

Thanks,
Vadym

**Comment by nagarajas** [comment]
Hi @VadymSV,

It is being recommended to move components from nuget package in this blog https://blog.xamarin.com/hello-nuget-new-home-xamarin-components/

Do you have any plans to publish it to nuget package repository?

**Comment by Bucimis** [comment]
Hi @VadymSV @nagarajas,

With the deprecation of the Xamarin Component Store, this is on our roadmap now. We will update with timelines as soon as we have them.

Thanks,
Braze SDK Engineering

**Comment by ATLRAM** [comment]
@Bucimis , any update as to when the nuget package will be deployed to the repository?

**Comment by Bucimis** [comment]
@ATLRAM @VadymSV @nagarajas @timahrentlov,

Thanks very much for your patience here. At long last, we've released our Xamarin Bindings to Nuget!

See:
https://www.nuget.org/packages/AppboyPlatform.AndroidBinding/
https://www.nuget.org/packages/AppboyPlatformXamariniOSBinding/

Notes:
1) We did not update the bindings for this release. The release available on Nuget is 1.5.0, same as the latest release in the Xamarin Component Store.
2) We will no longer be releasing to the Xamarin Component Store. 1.5.0 was our last release there.
3) External documentation is still forthcoming (but you already know what to do).

Thanks,
Braze SDK Engineering
